### DISCUSSION TITLE
Aprendendo sobre flexbox

### DISCUSSION THREAD

**Discussion by tcelestino** [frontendbr/forum#80] [Perguntas e Respostas]
Para quem quer aprender sobre o flexbox se divertindo:
- [Flexbox Froggy](http://flexboxfroggy.com/)
- [Flexbox Defense](http://www.flexboxdefense.com/)

**Comment by felipefialho** [comment]
Valeu @tcelestino, interessante pra carario.

Salvei os links!

**Comment by simaodeveloper** [comment]
Guia retirado do site CSS Tricks.
- [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

**Comment by dleitee** [comment]
Muito bom o Flexbox Defense!!!

**Comment by afonsopacifer** [comment]
Flexbox Froggy e Flexbox Defense são grandes exemplos de como poderiamos ensinar outras coisas tbm.. tipo fazer uma gamificação dessas para acessibilidade por exemplo... voltando ao assunto rsrsr, tem uma listinha com muita coisa legal no [awesome flexbox](https://github.com/afonsopacifer/awesome-flexbox) tbm ;)

**Comment by baltazarparra** [comment]
Tem esse curso também, http://flexbox.io/ o melhor e mais didático que já li, sobre flexbox.

**Comment by tcelestino** [comment]
@simaodeveloper, esse guia do CSS Tricks é o que mais abro quando fico na dúvida.

**Comment by leandroruel** [comment]
@tcelestino eu também uso esse guia do CSS Tricks, me salvou muitas vezes, como ainda é uma coisa "nova" de certa forma, sempre consulto o guia, ja tomei bronca por usar display: block; dentro de um elemento flex, e tive uns probleminhas com imagens responsivas que não escalavam em elementos flex também.

**Comment by felipefialho** [comment]
Esse guia do CSS Tricks fica aberto o tempo todo quando estou brincando com Flexbox :smiley_cat:

**Comment by tcelestino** [comment]
Reativando o tópcio só para deixar esse link: http://www.sketchingwithcss.com/samplechapter/cheatsheet.html

**Comment by laerciodev** [comment]
Tem o vídeo do nosso amigo @afonsopacifer no YouTube também:
https://www.youtube.com/watch?v=qBxGXb4iTQw

**Comment by paulinhoerry** [comment]
Cara, curti muito esse Flexbox Froggy ❤️

**Comment by adeonir** [comment]
Bem bacana mesmo esse Froggy

**Comment by icaromh** [comment]
Tem [esse tutorial](http://webdesign.tutsplus.com/pt/tutorials/how-to-build-a-news-website-layout-with-flexbox--cms-26611) de layout com Flexbox do tutsplus (que eu traduzi :hamster:)

**Comment by tarciozemel** [comment]
Mais um pt-br: http://desenvolvimentoparaweb.com/css/flexbox/

**Comment by cmpereirasi** [comment]
haha muito massa! Pessoal pirando no Flexbox ❤️ 
Alguém já utilizou o **Flexbox** para **produto final?**

**Comment by afonsopacifer** [comment]
@cmpereirasi Em todos os projetos brother, todos ❤️

**Comment by dleitee** [comment]
@cmpereirasi idem ao @afonsopacifer to usando bastante aqui em produção.

**Comment by cmpereirasi** [comment]
Show! alguém poderia **compartilhar** algo já pronto 😄 .

**Comment by afonsopacifer** [comment]
Alguns projetos que tem flexbox no meio:

http://afonsopacifer.com/index.html
http://afonsopacifer.github.io/open-source-checklist/
http://afonsopacifer.github.io/react-pomodoro/
http://afonsopacifer.github.io/react-comments/
http://afonsopacifer.github.io/css-magic/
http://afonsopacifer.github.io/egg-emergency/

Todos estão no Github ❤️ 

Ps: Tenho alguns projetos tipo sites ou apps mais focados em "negocio", mas isso é chato, a graça são os projetos open ❤️

**Comment by cmpereirasi** [comment]
Show @afonsopacifer

**Comment by simaodeveloper** [comment]
@tcelestino mano só vi agora que falou comigo, sim eu sempre volto, também, é difícil decorar quando não utilizamos sempre por causa do suporte para navegadores nos projetos ;)
